bot/exts/fun/uwu.py

```diff
@@ -53,6 +57,32 @@
 REGEX_NYA = re.compile(r"n([aeou][^aeiou])")
 SUBSTITUTE_NYA = r"ny\1"
 
+REGEX_EMOJI = re.compile(r"<(a?)?:(\w+):(\d{18})>?", re.ASCII)
```

Comment:
```suggestion
REGEX_EMOJI = re.compile(r"<(a)?:(\w+?):(\d{15,21}?)>", re.ASCII)
```
The `?` inside the first match (a?) is unneeded as the is another ? outside the match block, doing the same thing.

The ID for the emoji is a snowflake, so it's actually valid between 15 and 21 characters.